### DISCUSSION TITLE
unresolved handleDeepLink extension

### DISCUSSION THREAD

**Discussion by lamtech58** [arkivanov/Decompose#768] [Q&A] [answered]
Which dependency do i need to have this : com.arkivanov.decompose.handleDeepLink

**Comment by arkivanov** [comment, marked as answer]
This function is available since version 3.0.0, no additional dependencies needed.

**Reply by lamtech58** [reply]
i have this but its unresolved : decompose = "3.0.0-alpha05"

**Reply by arkivanov** [reply]
Please update to version 3.0.0. Version 3.0.0-alpha05 is older than 3.0.0.

**Reply by lamtech58** [reply]
oh im always confused about alpha beta thing! thanks
